docs/content/latest/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md

```diff
@@ -39,31 +39,21 @@ Use Yugabyte Platform to back up your YugabyteDB universe YSQL data.
 
 To schedule backups for a later time, or as a recurring task, see [Schedule universe YSQL data backups](../../schedule-data-backups/ysql).
 
-To immediately back up your YugabyteDB universe YSQL data:
+To immediately back up your YugabyteDB universe YSQL data, perform the following:
 
-1. Open the **Universe Overview** and then click the **Backups** tab. The **Backups** page appears.
+1. Open your universe and select **Backups**.
 1. Click **Create Backup** to open the **Create Backup** dialog.
 
     <br/><br/>
 
-    ![Create Backup - YSQL](/images/yp/create-backup-ysql.png)
+    ![Create Backup - YSQL](/images/yp/create-backup-ysql.png)<br><br>
 
-1. Click the **YSQL** tab and enter the following information:
+1. Complete the fields presented in the **YSQL** tab.
 
-    - **Storage**: Select the storage type: `GCS Storage`, `S3 Storage`, or `NFS Storage`.
-    - **Namespace**: Select the namespace from the drop-down list of available namespaces.
-    - **Parallel Threads**: Enter or select the number of threads. The default is `8`.
+1. Click **OK** to start the requested backup immediately.
 
-1. Click **OK**. The requested backup begins immediately.
+If the universe has [encryption at rest enabled](../../../security/enable-encryption-at-rest), data files are backed up as-is (encrypted) to reduce the computation cost of a backup and to keep the files encrypted. A universe key metadata file, containing key references, is also backed up. To allow Yugabyte Platform to back up your data with the user authentication enabled, follow the instructions provided in [Edit configuration flags ](../../../manage-deployments/edit-config-flags) to add the `ysql_enable_auth=true` and `ysql_hba_conf_csv="local all all trust"` YB-TServer flags.
```

Comment:
Are we removing notes in general from the docs? They looks good to underline edge-cases.

Reply:
Notes are still useful, but sometimes their are not used correctly (e.g. they are not meant to describe corner cases). In addition, our current css give the Note element too much prominence. It's on the doc team To Do list to redesign Notes, Warnings, Caution, etc.